src/scripts/main.js

```diff
@@ -1,3 +1,36 @@
 'use strict';
 
-// write your code here
+const populationList = document.querySelectorAll('.population');
+const populationArrey = [];
+
+for (let i = 0; i < populationList.length; i++) {
+  let item = document.querySelectorAll('.population')[i].textContent;
+
+  populationArrey.push(item = +item.replaceAll(',', ''));
+}
+
+let totalPopulation = 0;
+
+populationArrey.map(sum => {
+  totalPopulation += sum;
+});
```

Comment:
It would be nice place to use reduce method
https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce